resources/js/Pages/Category/Create.js

```diff
@@ -8,6 +8,7 @@ import { createCategory } from "../../Api";
 export default function Create({showCreate, onClose, onCreate}) {
     const [name, setName] = useState('')
     const [type, setType] = useState('EXPENSES')
+    const [color, setColor] = useState('')
```

Comment:
can we provide a default color to the user?

```suggestion
    const [color, setColor] = useState('badge-gray')
```